anitya/lib/backends/github.py

```diff
@@ -88,8 +88,8 @@ def get_versions(cls, project):
             (owner, repo) = url.split('/')
         except ValueError:
             raise AnityaPluginException(
-                'Project %s was incorrectly set-up. ' % project.name +
-                'Can\'t parse owner and repo.')
+                """Project {} was incorrectly set-up.
+                Can\'t parse owner and repo.""".format(project.name))
```

Comment:
FYI you can do
```
raise AnityaPluginException(
    "..."
    "more text".format())

Reply:
Didn't know that, it can't be done with `%`.